design/lpar-status.md

```diff
@@ -212,73 +259,41 @@ book.
-| Start                                       | operating        | ?                                    | No          |
-| Reset Normal/Clear                          | operating        | ?                                    | No          |
-| Reset Normal/Clear                          | not-operating    | ?                                    | No          |
+   However, the property description includes this statement:
 
-### Linux-type LPAR with auto-loading
+   "The 'group-uri' query parameter can be used on a 'Get Logical Partition
+   Properties' operation to specify the object URI of the Custom Group object
+   used for determining the next activation profile name to be used. If not
+   specified, the system-defined Logical Partition group is used for this
+   determination."
 
-Image Profile:
-* load-at-activation = True
-* ipl-type = ipl-type-standard
+   Questions:
 
-LPAR:
-* activation-mode = linux
+   - Does the specification of a group-uri query parameter on a Get Logical
+     Partition Properties operation influence the 'next-activation-profile-name'
+     property that is returned?
 
-Same as without auto-loading, except that activation is followed by a load:
+   - If so, what does it mean for the 'next-activation-profile-name' property
+     to be writeable?
 
-| LPAR operation                              | Status before    | Expected status after                | Verified    |
-|:------------------------------------------- |:---------------- |:------------------------------------ |:----------- |
-| Activate w/ image profile                   | not-activated    | operating                            | No          |
-| Activate w/ load profile                    | not-activated    | operating                            | No          |
-| Activate w/ image profile (unchanged)       | not-operating    | operating (no reactivation)          | No          |
-| Activate w/ load pr. (image pr. unchanged)  | not-operating    | operating (no reactivation)          | No          |
-| Activate w/ image profile (modified)        | not-operating    | operating (reactivation)             | No          |
-| Activate w/ force (image+load pr. unchanged)| operating        | operating (no react./no reload)      | No          |
-| Activate w/ force (image pr. modified)      | operating        | operating (react+reload)             | No          |
-| Activate w/ force (load pr. modified)       | operating        | operating (react+reload)             | No          |
+   - What is the "system-defined Logical Partition group"?
 
-### SSC-type LPAR
+2. 'load-type' parameter of activate operation for SSC/zAware partitions
 
-Same as Linux-type: Yes?
+   The new 'load-type' parameter of the "Activate Logical Partition" operation
+   in HMC>=2.16 is silent about whether it can be used for LPARs whose
+   image activation profile specifies 'activation-mode' as 'ssc' or 'zaware'.
```

Comment:
It cannot be used if the image's profile's activation-mode is ssc or zaware; there's a check to prevent it synchronously. It returns a bad request reason code 306

Reply:
Thanks for the clarification. I'll update the document.